packages/@aws-cdk/aws-glue/lib/job-executable.ts

```diff
@@ -287,7 +287,7 @@ export class JobExecutable {
       if (config.language !== JobLanguage.PYTHON) {
         throw new Error('Python shell requires the language to be set to Python');
       }
-      if ([GlueVersion.V0_9, GlueVersion.V1_0].includes(config.glueVersion)) {
+      if ([GlueVersion.V0_9].includes(config.glueVersion)) {
```

Comment:
If we know that Glue v2 is not allowed here, let's throw an error. Yes, it will take work to unblock, but I'd rather that very small effort plus delay rather than letting our customers believe that v2 is supported when it is not

Reply:
I've updated that